cypress_test/integration_tests/common/helper.js

```diff
@@ -513,9 +508,8 @@ function matchClipboardText(regexp) {
 	doIfInCalc(function() {
 		cy.cGet('body').contains('#copy-paste-container pre', regexp).should('exist');
 	});
-	doIfInImpress(function() {
-		cy.cGet('body').contains('#copy-paste-container pre', regexp).should('exist');
-	});
+
+	// FIXME: above is async, do a barrier
```

Comment:
Yes, this removes an Impress-specific assert and adds no replacement, as far as I see; but you did mark it with a FIXME.